src/tools/idyntree-model-view.cpp

```diff
@@ -0,0 +1,63 @@
+/**
+ * Copyright (C) 2016 Fondazione Istituto Italiano di Tecnologia
+ * Authors: Silvio Traversaro
+ * CopyPolicy: Released under the terms of the LGPLv2.1 or later, see LGPL.TXT
+ */
+
+#include <iDynTree/Model/Model.h>
+#include <iDynTree/ModelIO/ModelLoader.h>
+
+#include <iDynTree/Visualizer.h>
+
+#include "cmdline.h"
```

Comment:
It is an header-only simple library to parse options from command line: https://github.com/robotology/idyntree/blob/master/src/tools/cmdline.h .